app/assets/stylesheets/components/_document-table.scss

```diff
@@ -0,0 +1,48 @@
+table.document-table {
+  table-layout: fixed;
+  width: 904px;
+
+  th {
+
+    &.doc-type-column {
+      width: 165px;
+    }
+
+    &.link-column {
+      width: 360px;
+    }
+
+    &.upload-date-column {
+      width: 129px;
+    }
+
+    &.notes-column {
+      width: 250px;
+    }
+  }
+
+  td {
+
+    &.document-notes {
+      color: $color-gold-accent;
+    }
```

Comment:
name-wise, I'm having trouble guessing that these are. I would assume that the notes column is not always yellow. Is this for warnings only?

Reply:
The notes column only contains warnings about large file size or uncommon file type. The mock explicitly titles this column "Notes".